### DISCUSSION TITLE
Welcome to Miser Project Discussions!

### DISCUSSION THREAD

**Discussion by orcmid** [orcmid/miser#35] [General]
## 👋 Welcome!
  Discussions as a place to connect with others taking an interest in the Miser Project. 
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by orcmid** [comment]
Now that Discussions (beta) are available, I am going to reclassify a number of issues that are actually discussions.  This will not change their issue/discussion numbers and we'll see how that works.

I will (re-)label existing issues as Questions or Discussion and then have them bulk-changed to Discussions.  This will sort things out much better.

**Comment by band** [comment]
Bill Anderson has been intrigued by the foundations of programming and how computing works since his first FORTRAN program written in 1979 (I think). I want to see how the Miser project helps present the issues and answers and the still-open questions.
